source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+field name exists.
+
+The following example searches the ``guitars`` collection for any documents in
+which the ``rating`` field exists.
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-exists-search
+   :end-before: // end-exists-search
+   :language: csharp
+   :dedent:
+
+The search returns the following documents:
+
+.. code-block:: json
+
+   { "_id" : 1, "make" : "Fender", "description" : "...", "establishedYear" : 1946, "in-stock" : true, "rating" : 9 }
+   { "_id" : 2, "make" : "Gibson", "description" : "...", "establishedYear" : 1902, "in-stock" : true, "rating" : 8 }
+   { "_id" : 3, "make" : "PRS", "description" : "...", "establishedYear" : 1985, "in-stock" : true, "rating" : 9 }
+   { "_id" : 5, "make" : "Ibanez", "description" : "...", "establishedYear" : 1957, "in-stock" : true, "rating" : 7 }
+
+
+To learn more about the ``equals`` operation, see the :atlas:`Atlas
+exists </atlas-search/exists>` guide.
+
+GeoShape
+~~~~~~~~
+
+Use the ``GeoShape`` method to search for documents in relation to a given
+geometry. When specifying the coordinates to search, longitude must be specified
+first, followed by latitude. Longitude values can be between ``-180`` and
+``180``, inclusive. Latitude values can be between ``-90`` and ``90``,
+inclusive.
+
+.. include:: /includes/atlas-search-support-note.rst
+
+Consider some documents in the ``guitars`` collection have added an
+``in-stock-location`` field. The changed documents in the collection now look as
+follows:
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "description": "...", "establishedYear": 1946, "in-stock": true, "in-stock-location":  { "type": "Point", "coordinates": [ -73.93615, 40.69791 ]}, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "description": "...", "establishedYear": 1902, "in-stock": true, "in-stock-location":  { "type": "Point", "coordinates": [ 47.6062, 122.321 ]}, "rating": 8 }
+
+The following example searches for all documents in which the
+coordinates in the ``in-stock-location`` field intersect with a specified
+polygon:
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-geoshape-search
+   :end-before: // end-geoshape-search
+   :language: csharp
+   :dedent:
+
+The search returns the following document:
+
+.. code-block:: json
+
+   { "_id" : 1, "make" : "Fender", "description" : "...", "establishedYear" : 1946, "in-stock" : true, "in-stock-location" : { "type" : "Point", "coordinates" : ["-73.93615", "40.69791"] }, "rating" : 9 }
+
+To learn more about the ``geoShape`` operation, see the :atlas:`Atlas
+geoShape </atlas-search/geoShape>` guide.
+
+GeoWithin
+~~~~~~~~~
+
+Use the ``GeoWithin`` method to search for documents in which the coordinates of
+their :manual:`GeoJSON </reference/geojson/>` field are within a specified
+geometry. You can search for points that are within a:
```

Comment:
I believe it is possible for a document to have more than one GeoJSON field, since the only requirements to make it a GeoJSON field are for it to be an object a `type` field and a `coordinates` field.
In the case of this search, you would just have to specify in your path parameter which GeoJSON field you want to use for the search

Reply:
If they can have more than one, would it be correct to say "...the coordinates of their specified GeoJSON field..."? Because the way it reads above, it sounds like they can only ever have one.

Reply:
Yeah, adding "specified" there is much better. Changed!